library/tinkerpop/src/main/java/uk/gov/gchq/gaffer/tinkerpop/GafferPopGraph.java

```diff
@@ -411,12 +412,11 @@ public Iterator<Vertex> vertices(final Object... vertexIds) {
                 .map(e -> (Vertex) e)
                 .iterator();
 
-        final int resultSize = IterableUtils.size(translatedResults);
-        if (getAll && resultSize >= getAllElementsLimit) {
-            LOGGER.warn("Result size is equal to configured limit ({}). Results may have been truncated", getAllElementsLimit);
+        if (getAll && IterableUtils.size(translatedResults) > getAllElementsLimit) {
+            LOGGER.warn(
```

Comment:
Should this ever happen? Is the size > limit the results haven't been truncated

Reply:
Good spot, I removed this section when debugging so just been added back slightly wrong